### PR TITLE
[Hotfix] Fix the Chinese characters in the code

### PR DESCRIPTION
**Pull request by jiamin13579**
## Why are the changes needed?

In this code snippet, "36524" represents a specific numerical value, which actually denotes the total number of days in four years. This is because in the Gregorian calendar, a leap year occurs every four years. Therefore, "36524" signifies the total number of days in four years.

## Brief change log

- Modify the Chinese characters to "36524".

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: amoro-mixed-format/amoro-mixed-format-flink/amoro-mixed-format-flink-common/src/main/java/org/apache/amoro/flink/util/DateTimeUtils.java [hunk 1]
```diff
@@ -958,8 +958,8 @@ private static int julianExtract(TimeUnitRange range, int julian) {
     int j = julian + 32044;
     int g = j / 146097;
     int dg = j % 146097;
-    int c = (dg / '躬' + 1) * 3 / 4;
-    int dc = dg - c * '躬';
+    int c = (dg / 1461 + 1) * 3 / 4;
```
Comment (zhongqishang) [inline amoro-mixed-format/amoro-mixed-format-flink/amoro-mixed-format-flink-common/src/main/java/org/apache/amoro/flink/util/DateTimeUtils.java:961]: I found it in the source file of the flink project as `36524`, not `1461`. Can you help confirm it again?

Comment (jiamin13579) [reply amoro-mixed-format/amoro-mixed-format-flink/amoro-mixed-format-flink-common/src/main/java/org/apache/amoro/flink/util/DateTimeUtils.java]: Thanks, I checked the flink code and it also has Chinese characters. But when I print the '躬' char and convert it to int, it is indeed 36524.

**Review by zhongqishang — APPROVED** [review]
LGTM. Thanks for your contribution.

**Review by tcodehuber — APPROVED** [review]
LGTM

**Pull request merged**